src/pages/page-not-found/page-not-found.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
```suggestion
```
We're using [React 17](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html), so no need to import react anymore!